modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -42,10 +43,20 @@ public static String[] getExpectedMessages() throws Exception {
 
 			String from = jsonObject.getString("from");
 
-			if (from.contains(StringPool.OPEN_PARENTHESIS) &&
-				!jsonObject.getBoolean("skipParametersValidation")) {
+			String[] validExtensions = JSONUtil.toStringArray(
+				jsonObject.getJSONArray("validExtensions"));
+
+			if (validExtensions.length==0) {
+				validExtensions = new String[] {"java"};
+			}
 
-				expectedMessages.add(_getMessage(jsonObject));
+			for (String validExtension : validExtensions) {
```

Comment:
@MicaelleSilvaa, do you really need to use this `for` like this?

How about replace this code with the following?
```
String[] validExtensions = JSONUtil.toStringArray(
	jsonObject.getJSONArray("validExtensions"));

if (validExtensions.length > 0 && !validExtensions.contains("java")) {
    continue;
}
```